server/build.gradle.kts

```diff
@@ -13,18 +16,22 @@ plugins {
 	jacoco
 }
 
+
 group = "pis-hello"
 version = "0.0.1-SNAPSHOT"
 java.sourceCompatibility = JavaVersion.VERSION_11
 
 repositories {
+	google()
 	mavenCentral()
 }
 
 dependencies {
+	implementation("androidx.camera:camera-camera2:1.0.0-alpha07")
 	val springBootVersion = "2.6.2"
 	val jacocoVersion = "0.8.7"
 	val sqliteJDBCVersion = "3.36.0.2"
+	val textRecognitionVersion = "17.0.1"
```

Comment:
Why do we have text recognition in our server dependencies?

Reply:
Dependencies deleted